reference/commands/custom_commands.rst

```diff
@@ -0,0 +1,139 @@
+.. _examples_extensions_custom_commands:
+
+Custom commands
+=================
+
+Since Conan 2.0, it's so easy to create your own Conan commands.
```

Comment:
I think you should change this sentence to something more meaningful, explaining why it's easy or remove it otherwise.

Reply:
I thought I already removed it. I don't think I can tell anything meaningful before explaining the full reference... Anyway, I'll try to think of something else.

Reply:
I think it's enough with a sentence that explains that Creating custom commands in Conan is as easy as adding your own modules in a folder and that you can use the Conan API, etc.
Also, I think there's no need to say that this is since Conan 2.0